lib/services/access_request_cache.go

```diff
@@ -186,6 +188,8 @@ func (c *AccessRequestCache) ListMatchingAccessRequests(ctx context.Context, req
 		index = accessRequestCreated
 	case proto.AccessRequestSort_STATE:
 		index = accessRequestState
+	case proto.AccessRequestSort_USER:
```

Comment:
should we add a test for this? there is a test in `auth/access_request_test` that seems to tests all sort types